tests/integration/remote/util.py

```diff
@@ -48,11 +48,25 @@ def get_gas_fees_for_remote() -> tuple:
 
     if not gas_resp or gas_resp.status_code != 200:
         print("Invalid response from Polygon gas station. Retry with brownie values...")
+        required_confs = 20
+        if chain.id == 80001:
```

Comment:
required_confs = 4 if chain.id == 80001 else 20

:)

Reply:
the chain ID needs to be changed then, because for polygon mainnet is 20 and it has a different chain ID.

So it would be:
required_confs = 4 if chain.id == 137 else 20